Makefile

```
@@ -23,6 +23,31 @@ bench-announce:
 bench-run:
 	@go test -bench=. ./...
 
+# Blocking performance guards: stable on shared runners, unlike latency.
+# Thresholds are predeclared in docs/improvements/thresholds.md.
+guards: guards-race guards-allocs
```

checkmake (0.3.2)

[warning] Target "guards" should be declared PHONY.

(phonydeclared)